config/routes.rb

```diff
@@ -4,13 +4,16 @@
   namespace :api, defaults: { format: 'json' } do
     namespace :v1 do
       resources :quizzes do
-        resources :questions, shallow: true
+        resources :questions, shallow: true  do
+          resources :answers
```

Comment:
I think you might want to make all resources `shallow`, otherwise to GET/UPDATE/DELETE an `Answer` you'll need to know which `Question` it belongs to and it shouldn't really be necessary (also, the frontend is implemented in such a way that it expects all routes to be shallow)

Reply:
I fixed the spelling mistakes and made all resources shallow. I hope it's ok now! :)